### PR TITLE
Make inspects valid Elixir expressions

### PR DESCRIPTION
**Pull request by mat-hek**

### CODE REVIEW

File: lib/bimultimap.ex [hunk 1]
```diff
@@ -643,11 +643,7 @@ defmodule BiMultiMap do
     import Inspect.Algebra
 
     def inspect(bimultimap, opts) do
-      concat([
-        "#BiMultiMap<",
-        Inspect.List.inspect(BiMultiMap.to_list(bimultimap), opts),
-        ">"
-      ])
+      concat(["BiMultiMap.new(", to_doc(BiMultiMap.to_list(bimultimap), opts), ")"])
```
Comment (mkaput) [inline lib/bimultimap.ex:646]: MapSet still uses `Inspect.List.inspect`: https://github.com/elixir-lang/elixir/blob/v1.14.0/lib/elixir/lib/map_set.ex#L506-L514

I don't know what's the difference, but I think we should still use old approach here to match Elixir's behaviour?

File: lib/bimap.ex [hunk 2]
```diff
@@ -621,7 +621,7 @@ defmodule BiMap do
     import Inspect.Algebra
 
     def inspect(bimap, opts) do
-      concat(["#BiMap<", Inspect.List.inspect(BiMap.to_list(bimap), opts), ">"])
+      concat(["BiMap.new(", to_doc(BiMap.to_list(bimap), opts), ")"])
```
Comment (mkaput) [inline lib/bimap.ex:624]: same here

File: lib/bimultimap.ex (hunk 1 above)
Comment (mat-hek) [reply lib/bimultimap.ex:646]: hmm, the docs here https://hexdocs.pm/elixir/1.14/Inspect.html#inspect/2 say that `to_doc` should be used :P

Comment (mat-hek) [reply lib/bimultimap.ex:646]: @mkaput bump

Comment (mkaput) [reply lib/bimultimap.ex:646]: Hmm, looking at the source code `to_doc` seems to be going to call `Inspect.List.inspect` anyway. Let's leave this as you wrote.

**Review by mkaput — APPROVED** [review]

**Pull request merged**